### PR TITLE
docs: translate to Spanish python-package-build-tools.md

### PR DESCRIPTION
**Pull request by RobPasMue**
Related to #313

### CODE REVIEW

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 1]
```diff
@@ -3677,33 +3944,41 @@ msgid ""
...
 msgid ""
 "Poetry creates a **poetry.lock** file that you can use if you need a lock"
 " file for your build."
 msgstr ""
+"Poetry crea un archivo **poetry.lock** que puede usar si necesita un archivo de bloqueo para su construcción."
 
 #: ../../package-structure-code/python-package-build-tools.md:392
 msgid "Poetry supports publishing to both test PyPI and PyPI"
-msgstr ""
+msgstr "Poetry admite la publicación tanto en test PyPI como en PyPI"
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:3964]:
```suggestion
msgstr "Poetry admite la publicación tanto en PyPI Test como en PyPI"
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 2]
```diff
@@ -3742,13 +4019,19 @@ msgid ""
 "follows: `poetry add \"requests>=2.1\"` See breakout below for more "
 "discussion on issues surrounding upper-bounds pinning."
 msgstr ""
+"Poetry, por defecto, fija las dependencias utilizando un límite de \"límite superior\" especificado con el "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:4022]:
```suggestion
"Poetry, por defecto, fija las dependencias utilizando un valor de \"límite superior\" de versión especificado con el "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 3]
```diff
@@ -3742,13 +4019,19 @@ msgid ""
...
+"símbolo `^` por defecto. Sin embargo, este comportamiento puede ser sobrescrito especificando la dependencia "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:4023]:
```suggestion
"símbolo `^` por defecto. Sin embargo, este comportamiento puede ser eliminado especificando la dependencia "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 4]
```diff
@@ -3771,6 +4057,11 @@ msgid ""
 "never bump the dependency to 2.0 even if there is a new major version of "
 "the package. Poetry will instead bump up to 1.9.x."
 msgstr ""
+"Por defecto, Poetry fija las dependencias utilizando `^` por defecto. Este símbolo `^` significa que hay un "
+"\"límite superior\" a la dependencia. Por lo tanto, Poetry no incrementará la versión de una dependencia a una "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:4061]:
```suggestion
"\"límite superior\" en la dependencia. Por lo tanto, Poetry no incrementará la versión de una dependencia a una "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 5]
```diff
@@ -3803,57 +4100,66 @@ msgid ""
...
+"frontend de usuario como Flit, Poetry y Hatch ofrecen. Como tal, necesitará usar otras herramientas como **build** "
+"para crear las distribuciones de su paquete y **twine** para publicar en PyPI."
 
 #: ../../package-structure-code/python-package-build-tools.md:448
 msgid ""
 "While setuptools is the most commonly used tool, we encourage package "
 "maintainers to consider using a more modern tool for packaging such as "
 "Poetry, Hatch or PDM."
 msgstr ""
+"Aunque setuptools es la herramienta más comúnmente utilizada, alentamos a los mantenedores de paquetes a considerar "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:4115]:
```suggestion
"Aunque setuptools es la herramienta más comúnmente utilizada, sugerimos a los mantenedores de paquetes a considerar "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 6]
```diff
@@ -3803,57 +4100,66 @@ msgid ""
...
+"usar una herramienta más moderna para el empaquetado como Poetry, Hatch o PDM."
 
 #: ../../package-structure-code/python-package-build-tools.md:451
 msgid ""
 "We discuss setuptools here because it's commonly found in the ecosystem "
 "and contributors may benefit from understanding it."
 msgstr ""
+"Discutimos setuptools aquí porque comúnmente se encuentra en el ecosistema y los contribuyentes pueden beneficiarse "
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:4123]:
```suggestion
"Discutimos setuptools aquí porque se encuentra muy a menudo en el ecosistema y entenderlo "
```

File: locales/es/LC_MESSAGES/package-structure-code.po [hunk 7]
```diff
@@ -3803,57 +4100,66 @@ msgid ""
...
+"de entenderlo."
```
Comment (flpm) [inline locales/es/LC_MESSAGES/package-structure-code.po:4124]:
```suggestion
"puede beneficiarle."
```

**Review by flpm — COMMENTED** [review]
That was a lot of translation, thanks for helping! Just added some minor rephrases in cases where the literal translation was not the best approach.

**Comment by RobPasMue** [comment]
Thanks a bunch @flpm for your thorough review and recommendations!

**Review by flpm — APPROVED** [review]
LGTM

**Pull request merged**